package-lock.json

```diff
@@ -180,7 +180,6 @@
         }
       ],
       "license": "MIT",
-      "peer": true,
```

Comment:
You keep deleting these lines with your dependency updates, probably because you are using an older version of npm? I doubt it matters for anything important, so I will just merge it, but maybe you would find it useful to update your npm version.